### ISSUE TITLE
will this support 3rd level domain selling?

### ISSUE THREAD

**Issue by anantsparrow** [internetee/registry#2596]
hi i was just willing , if this software can support selling of 3rd level domains like it.com is doing

**Comment by vohmar** [comment]
Yes our software can be set up to manage domain registrations at any level
